### ISSUE TITLE
Incorrect dependencies

### ISSUE THREAD

**Issue by DManowitz** [conda-forge/qp-feedstock#18]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

As seen by the [requirements.txt](https://github.com/LSSTDESC/qp/blob/v0.8.1/requirements.txt) file on the qp repo, this package requires scipy >=1.9.0 (and thus, python >=3.8).  This package on conda-forge reports requirements of scipy >=1.7.0 and python >=3.6, which is incorrect.

### Installed packages

```shell
N/A
```

### Environment info

```shell
N/A
```

**Comment by eacharles** [comment]
Yes conda-forge is a bit of a pain b/c it doesn’t take the dependencies from pyproject.toml.On May 21, 2023, at 1:50 PM, David Manowitz ***@***.***> wrote:﻿
Solution to issue cannot be found in the documentation.

 I checked the documentation.

Issue
As seen by the requirements.txt file on the qp repo, this package requires scipy >=1.9.0 (and thus, python >=3.8).  This package on conda-forge reports requirements of scipy >=1.7.0 and python >=3.6, which is incorrect.
Installed packages
N/A
Environment info
N/A
